src/plugin.rs

```diff
@@ -225,6 +257,11 @@ impl<A: Actionlike> Default for ToggleActions<A> {
 /// `Reset` must occur before `Update`
 #[derive(SystemSet, Clone, Hash, Debug, PartialEq, Eq)]
 pub enum InputManagerSystem {
+    /// Performs actions before [`InputManagerSystem::Update`]
+    ///
+    /// This is useful for temporarily disabling [`TrackingInputType`],
+    /// and it will be re-enabled after handling all conflicting inputs
+    PreUpdate,
```

Comment:
This is a confusing name: it conflicts with the `Schedule`.

Reply:
Remove this by adding `TrackingState` enum